protocol/activity_center.go

```diff
@@ -63,12 +63,16 @@ type ActivityCenterPaginationResponse struct {
 	Notifications []*ActivityCenterNotification `json:"notifications"`
 }
 
+type ActivityCenterState struct {
+	Unread  uint64 `json:"unread"`
```

Comment:
@MishkaRogachev, just a heads up, this `Unread` count won't be used by mobile at all, because we need to track the count of notifications by type, given that every tab in the Activity Center needs to display its own *unread indicator*.

As of now, the solution on mobile has been to call the endpoint `UnreadAndAcceptedActivityCenterNotificationsCount` for each type, which accepts a notification type as a filter. It's not ideal, but gets the job done. It'd be even better to get a single endpoint on status-go providing counts for multiple notification types so we can reduce the overhead of doing multiple RPC calls.

Reply:
@ilmotta @cammellos 
(Total) unread count and seen i use for this bell icon. Isn't there need to show same in mobile client?
<img width="83" alt="Screenshot 2023-02-15 at 12 35 58" src="https://user-images.githubusercontent.com/2522130/218975281-6ea14550-60da-4868-9098-f42d010eadc8.png">

About `UnreadAndAcceptedActivityCenterNotificationsCount` i also can add it for each activity category

Reply:
@MishkaRogachev, there's certainly a need, but the value is **derived** (summed up) from a different state that's tracking the unread count of different types.